js_modules/dagster-ui/packages/app-oss/src/extractInitializationData.ts

```diff
@@ -30,5 +30,10 @@ export const extractInitializationData = (): {
       }
     }
   }
+  if (value.telemetryEnabled) {
```

Comment:
testing this with telemetry on and off is important

Reply:
yeah this path has been tested for the existing telemetry stuff but I can double check